challenges/backend/app.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, request, abort, jsonify
+import json
+
+# The file in which the numbers get saved to in JSON format.
+# This way if the server happens to go down the data will persist.
+STORED_FILE_NAME = 'numbers.txt'
+
+# The number of numbers we expect the POST request to have.
+ARRAY_LENGTH = 500
+
+app = Flask(__name__)
+
+# Helper function to write data to our stored file
+#
+# numbers - list of numbers to be written
+def write_numbers(numbers):
+  with open(STORED_FILE_NAME, 'w') as storedFile:
+    json.dump(numbers, storedFile)
+
+# Helper function to write data to our stored file
+#
+# returns - list of numbers read from file
+def read_numbers():
+  with open(STORED_FILE_NAME, 'r') as storeFile:
+    return json.load(storeFile)
+    
+  # If we cannot return the json then we could not open the file
+  raise Exception('Could not read file correctly.')
```

Comment:
Here raising an Exception also works but in general it is better practice to raise a specific Error.